docker/Dockerfile.distroless

```diff
@@ -1,5 +1,5 @@
 # prepare a distroless source context to copy files from
-FROM gcr.io/distroless/static-debian11@sha256:a01d47d4036cae5a67a9619e3d06fa14a6811a2247b4da72b4233ece4efebd57 as distroless_source
+FROM gcr.io/distroless/static-debian11@sha256: as distroless_source
```

Comment:
Just to clarify, this is definitely wrong. Should be 7198a357ff3a8ef750b041324873960cf2153c11cc50abb9d8d5f8bb089f6b4e.

I am guessing you are missing `crane`

Reply:
I've also seen the problem using Docker Desktop for Mac in some versions where their VM doesn't like `sed`. You can look for `sed` related message when running the code. I'm fixed in in some cases by picking a file system other than VirtioFS.